README.md

```diff
@@ -9,35 +9,21 @@ LEGO Universe server written in C#
 Contributions are always welcome! Feel free to open pull requests or issues to help with the continued development of Uchu.
 
 ## Python scripting
-Uchu supports [Python scripting](https://github.com/yuwui/Uchu/blob/master/Uchu.Python/SCRIPTING.md), which you can use to code minigames, new game features, and a lot more! This is the perfect way for those of you with less programming experience to contribute.
+Uchu supports [Python scripting](https://github.com/UchuServer/Uchu/blob/master/Uchu.Python/SCRIPTING.md), which you can use to code minigames, new game features, and a lot more! This is the perfect way for those of you with less programming experience to contribute.
 
 ## Releases
-Check out the [release page](https://github.com/yuwui/Uchu/releases) for standalone binary releases of Uchu.
+Check out the [release page](https://github.com/UchuServer/Uchu/releases) for standalone binary releases of Uchu.
 
 ## Prerequisites
 This project will be built and run using a CLI. You will also require a text editor capable of editing raw text files. These are requirements of setting up this project so if you are not familiar with either it is recommended to research them in advance.
 Users should also understand that Uchu is not a game in and of itself, but a server which tells a program on your computer - the client - what to do.
 Uchu is built with .NET Core 3.1 (see below for installation details) which is compatible with Windows, Linux, and MacOS.
 
-### .NET Core 3.1
-Install .NET Core 3.1 SDK for your OS using the instructions found on [here](https://dotnet.microsoft.com/download/dotnet-core/3.1).
+### .NET Core 5.0
```

Comment:
Minor thing is that .NET 5.0 is the correct name, not .NET Core 5.0. Microsoft is trying to move to a single .NET instead of .NET Framework vs .NET Core.

Reply:
Wrote this correctly in other places but forgot to change the readme. Also caught a leftover "3.1" in there. Thanks :+1: